tests/test_attention.py

```diff
@@ -1,9 +1,15 @@
 import torch
 
+from vformer.attention.cross import CrossAttention
 from vformer.attention.spatial import SpatialAttention
 from vformer.attention.vanilla import VanillaSelfAttention
 from vformer.attention.window import WindowAttention
 
+test_tensor1 = torch.randn(2, 65, 1024)
```

Comment:
I believe this reverts some of the changes @abhi-glitchhg made in #23? Could you check?